Scripts/travis.sh

```diff
@@ -27,7 +27,7 @@ ACTION="test"
 #  $1 : .xcodeproj file
 #  $2 : scheme to run
 #
-# The output is prettified using xcpretty and redirected to xcodebuild.log for failure analysis.
+# The output is formatted using xcpretty and redirected to xcodebuild.log for failure analysis.
 execute_xcodebuild() {
   if [ -z ${1+x} ]; then
```

Comment:
Remove x (also below):
if [ -z ${1} ]; then
is what you need, this probably worked because unset x is being treated as 0.